common/service/dynamicconfig/constants.go

```diff
@@ -35,11 +35,15 @@ func (k Key) String() string {
 var keys = map[Key]string{
 	unknownKey: "unknownKey",
 
-	testGetPropertyKey:         "testGetPropertyKey",
-	testGetIntPropertyKey:      "testGetIntPropertyKey",
-	testGetFloat64PropertyKey:  "testGetFloat64PropertyKey",
-	testGetDurationPropertyKey: "testGetDurationPropertyKey",
-	testGetBoolPropertyKey:     "testGetBoolPropertyKey",
+	testGetPropertyKey:                         "testGetPropertyKey",
+	testGetIntPropertyKey:                      "testGetIntPropertyKey",
+	testGetFloat64PropertyKey:                  "testGetFloat64PropertyKey",
+	testGetDurationPropertyKey:                 "testGetDurationPropertyKey",
+	testGetBoolPropertyKey:                     "testGetBoolPropertyKey",
+	testGetIntPropertyFilteredByDomainKey:      "testGetIntPropertyFilteredByDomainKey",
+	testGetDurationPropertyFilteredByDomainKey: "testGetDurationPropertyFilteredByDomainKey",
+
+	EnableGlobalDomainKey: "enableGlobalDomainKey",
```

Comment:
May want to use prefix of "system."

Either remove Key suffix or use Key for all other properties.